include/dpp/integration.h

```diff
@@ -169,5 +169,5 @@ typedef std::unordered_map<snowflake, integration> integration_map;
 /** A group of connections */
 typedef std::unordered_map<snowflake, connection> connection_map;
 
-};
+} // namespace dpp
```

Comment:
are these really needed when every sane IDE shows what namespace the curly braces belong to?
seems to have made the pr huge.

Reply:
well seems like the point of the PR is just "tidy up" stuff according to their coding style

Reply:
That's a bit unfair, I'd say. Aside from the closing namespace comments, these are common modern practices really, with clear improvements and no drawbacks.